lib/prometheus_exporter/client.rb

```diff
@@ -2,6 +2,7 @@
 
 require "socket"
 require "logger"
+require "openssl"
```

Comment:
in web server we delay require it, I guess we should do the same here?

Reply:
@SamSaffron done!